config/account.go

```diff
@@ -39,6 +39,7 @@ type Account struct {
 	PriceFloors             AccountPriceFloors                   `mapstructure:"price_floors" json:"price_floors"`
 	Validations             Validations                          `mapstructure:"validations" json:"validations"`
 	DefaultBidLimit         int                                  `mapstructure:"default_bid_limit" json:"default_bid_limit"`
+	Privacy                 AccountPrivacy                       `mapstructure:"privacy" json:"privacy"`
```

Comment:
In [GPP Phase 4](https://github.com/prebid/prebid-server/issues/2686) `allowactivities` seems to be inside a `consent` field. Should we rename to `consent` instead? 
```
 40       Validations             Validations                          `mapstructure:"validations" json:"validations"`
 41       DefaultBidLimit         int                                  `mapstructure:"default_bid_limit" json:"default_bid_limit"`
 42 -     Privacy                 AccountPrivacy                       `mapstructure:"privacy" json:"privacy"`
    +     Consent                 AccountPrivacyConsent                `mapstructure:"consent" json:"consent"`
 43   }
 config/account.go
```

Reply:
The Phase 4 docs are outdated. We changed it to "privacy" in the Phase 3 doc. I'm sure Phase 4 will be updated soon.

Reply:
Fixed in the referenced issue.